client/gatsby-config.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-
+const config = require('./config/env');
```

Comment:
```suggestion
const config = require('./config/env');

```

Nitpick on formatting.